src/Server/RequestHandler/AdvancedStaticFilesServer.php

```diff
@@ -20,72 +21,9 @@
 final class AdvancedStaticFilesServer implements RequestHandlerInterface, BootableInterface
 {
     /**
-     * Default static file extensions supported.
-     *
-     * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types
+     * @var array<string,string>
      */
-    private const FILE_EXTENSION_MIME_TYPE_MAP = [
```

Comment:
I like the changes overall, but this would be a major bc break if not used without Symfony. I would leave that "default" list, and allow to provide overrides what do you think? 

simple `array_merge` like this would be enough 

```
  private $mimeTypes;

  public function __construct(
        RequestHandlerInterface $decorated,
        HttpServerConfiguration $configuration,
        array $mimeTypesOverride = []
    ) {
        $this->decorated = $decorated;
        $this->cachedMimeTypes = [];
        $this->configuration = $configuration;
        $this->mimeTypes = array_merge(self:: FILE_EXTENSION_MIME_TYPE_MAP, $mimeTypesOverride);
        $this->cachedMimeTypes = [];
    }



```

Reply:
is there a possibility that the class will be used outside of symfony scope when we're  in a symfony bundle? I don't mind leaving the default list as it was an override it when new non-empty list gets into the constructor, but I don't see the point in that. your call, I can live with it bothways :)

Reply:
I'm using it sometimes without symfony. :)

Reply:
okay then, I'll make the changes